## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 482629`
Commit: `6a42441029b95e4e7b58326dc9af04ad57c80e09`

---
### `x86_64-darwin`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>acl2</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>abc-verifier</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>acl2</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>abc-verifier</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>acl2</summary>
<pre>   | ******** FAILED ********
   | ; (EV-REC *RETURN-LAST-ARG3* ...) took 
   | ; 89.05 seconds realtime, 0.46 seconds runtime
   | ; (137,831,504 bytes allocated).
   | ACL2 !&gt;Bye.
   | Exit code from ACL2 is 0
   | date.cert seems to be missing

**CERTIFICATION FAILED** for /nix/store/65b41m684sbj8mg5fag1y1z3nfnyjkgp-acl2-8.6/share/acl2/books/oslib/tests/date.lisp

make: *** [/nix/store/65b41m684sbj8mg5fag1y1z3nfnyjkgp-acl2-8.6/share/acl2/books/build/make_cert:116: oslib/tests/date.cert] Error 1
make: *** Attente des tâches non terminées....
Making /nix/store/65b41m684sbj8mg5fag1y1z3nfnyjkgp-acl2-8.6/share/acl2/books/projects/apply/report.cert on 31-Jan-2026 00:29:47.444
Built /nix/store/65b41m684sbj8mg5fag1y1z3nfnyjkgp-acl2-8.6/share/acl2/books/projects/apply/prog-mode-tests-book.cert (1.138s) at 31-Jan-2026 00:29:48.545
Built /nix/store/65b41m684sbj8mg5fag1y1z3nfnyjkgp-acl2-8.6/share/acl2/books/projects/apply/report.cert (1.463s) at 31-Jan-2026 00:29:48.915
Built /nix/store/65b41m684sbj8mg5fag1y1z3nfnyjkgp-acl2-8.6/share/acl2/books/models/jvm/m1/tmi-reductions.cert (94.319s) at 31-Jan-2026 00:29:48.964
Built /nix/store/65b41m684sbj8mg5fag1y1z3nfnyjkgp-acl2-8.6/share/acl2/books/projects/apply/loop-tests.cert (3.268s) at 31-Jan-2026 00:29:50.192
Built /nix/store/65b41m684sbj8mg5fag1y1z3nfnyjkgp-acl2-8.6/share/acl2/books/parsers/earley/grammar-reader.cert (93.037s) at 31-Jan-2026 00:29:51.102
Built /nix/store/65b41m684sbj8mg5fag1y1z3nfnyjkgp-acl2-8.6/share/acl2/books/kestrel/crypto/salsa/salsa20-tests.cert (130.621s) at 31-Jan-2026 00:30:19.933
Built /nix/store/65b41m684sbj8mg5fag1y1z3nfnyjkgp-acl2-8.6/share/acl2/books/system/fmt-support.cert (1852.015s) at 31-Jan-2026 00:56:55.090</pre>
</details>
</details>

--- a/pkgs/by-name/ab/abc-verifier/package.nix
+++ b/pkgs/by-name/ab/abc-verifier/package.nix
@@ -8,17 +8,22 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "abc-verifier";
-  version = "0.55";
+  version = "0.61";
 
   src = fetchFromGitHub {
     owner = "yosyshq";
     repo = "abc";
     rev = "v${finalAttrs.version}";
-    hash = "sha256-Ib6bZSPQmpI1UOsUG733TH6W6v+UnLyagdjUc8MreKw=";
+    hash = "sha256-j/JmEJa29mT/XQmVL/adVbj4S+AAgpga1jbEinCN16w=";
   };
 
   nativeBuildInputs = [ cmake ];
   buildInputs = [ readline ];
+
+  cmakeFlags = [
+    # This prevents CMake from trying to download googletest during the build
+    (lib.cmakeBool "ABC_SKIP_TESTS" true)
+  ];
 
   installPhase = ''
     runHook preInstall
@@ -30,7 +35,7 @@ stdenv.mkDerivation (finalAttrs: {
   passthru.rev = finalAttrs.src.rev;
 
   meta = {
-    description = "Tool for squential logic synthesis and formal verification";
+    description = "Tool for sequential logic synthesis and formal verification";
     homepage = "https://people.eecs.berkeley.edu/~alanmi/abc";
     license = lib.licenses.mit;
     maintainers = with lib.maintainers; [

--- a/pkgs/development/interpreters/acl2/default.nix
+++ b/pkgs/development/interpreters/acl2/default.nix
@@ -3,6 +3,7 @@
   stdenv,
   callPackage,
   fetchFromGitHub,
+  fetchpatch,
   runCommandLocal,
   makeWrapper,
   replaceVars,
@@ -50,6 +51,12 @@ stdenv.mkDerivation rec {
   libipasir = callPackage ./libipasirglucose4 { };
 
   patches = [
+    # The upstream fix for the input-files macro regression
+    (fetchpatch {
+      url = "https://github.com/acl2/acl2/commit/be39e7835f1c68008c17188d2f65eeaef61632fa.patch";
+      sha256 = "sha256-pZ/r0vlyJz7ymYfrVtHDxsLdw0M/MJStBH42ZLO7Fs4=";
+    })
+
     (replaceVars ./0001-path-changes-for-nix.patch {
       libipasir = "${libipasir}/lib/${libipasir.libname}";
       libssl = "${lib.getLib openssl}/lib/libssl${stdenv.hostPlatform.extensions.sharedLibrary}";

